README.md

```diff
@@ -50,6 +57,20 @@ We expect teams to automate testing, deployment and monitoring of their features
 - **Higher Quality** - Bringing engineers closer to the day-to-day operation of our business establishes a direct feedback loop with our users and allows for faster iteration and ultimately higher quality software design.
 - **Accountability** - Being responsible for operations for the code we write is an incentive to do things better. Nobody wants to wake up during the night to fix a memory leak in production.
 
+## Show, don’t tell: Deliver working software early and frequently.
+
+We believe in the basic Agile principle of "Release early and release often". This approach helps us manage risk and make better decisions by creating fast feedback loops between ourselves and our users.
+
+While prototypes and proof-of-concepts can be great tools early on, we should always maintain shippable code. Avoid creating technical debt. Don't depend on polish happening later. 
+
+Gather feedback where uncertainty exists. We are our own best testing team and as software professionals we shouldn't let our users remind us what quality software looks like. Don't ship and apologise for incomplete or broken features. Ship [minimum lovable products](https://productschool.com/blog/product-management-2/minimum-lovable-product/) instead.
+
+### Why?
+
+- **Code wins arguments**: Debating abstract ideas leads to endless meetings. A working piece of software is always better than architecture diagrams and RFCs.
+- **Move fast**: Putting feedback loops in place early on helps us make more informed decisions collaboratively and commit to deliverables that create tangible progress.
+- **Risk management**: Even with thorough design work and great planning, unplanned work is guaranteed to appear once you're in the process of building. So better start early. 
```

Comment:
```suggestion
- **Risk management**: Even with throughout design work and great planning, unplanned work is guaranteed to appear once you're in the process of building. So better start early. 
```

Reply:
Little typo.

Reply:
Actually not a typo.

I meant [thorough](https://dictionary.cambridge.org/dictionary/english/thorough) as in careful. 

But maybe for simpler language careful would be better 🙌